Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.csproj

```diff
@@ -8,7 +8,7 @@
 	  <AssemblyName>Frends.SFTP.UploadFiles</AssemblyName>
 	  <RootNamespace>Frends.SFTP.UploadFiles</RootNamespace>
 
-	  <Version>2.5.5</Version>
+	  <Version>2.5.6</Version>
```

Comment:
Since there's a modification to the result object, should there be an update to the minor version instead of the patch? (2.6.0)